writing-apps/creating-layouts.md

```diff
@@ -120,22 +109,26 @@ hello_button.clicked.connect (() => {
     hello_button.sensitive = false;
 });
 
-rotate_button.clicked.connect (() => {
-    rotate_label.angle = 90;
-    rotate_label.label = _("Vertical");
-    rotate_button.sensitive = false;
+swap_button.clicked.connect (() => {
+    grid.remove (hello_label);
+    grid.remove (swap_label);
+
+    grid.attach_next_to (hello_label, swap_button, Gtk.PositionType.RIGHT, 1, 1);
+    grid.attach_next_to (swap_label, hello_button, Gtk.PositionType.RIGHT, 1, 1);
+
+    swap_label.label = _("Swapped");
+    swap_button.sensitive = false;
```

Comment:
There is no `angle` property anymore, so instead I swap the labels. I think it's fine since it also shows how to remove widgets from the grid.